lib/openapi3.js

```diff
@@ -427,40 +427,58 @@ function getResponses(data) {
     for (let r in data.operation.responses) {
         if (!r.startsWith('x-')) {
             let response = data.operation.responses[r];
-            let entry = {};
-            entry.status = r;
-            entry.meaning = (r === 'default' ? data.translations.responseDefault : data.translations.responseUnknown);
+
+            let meta = {};
+            meta.meaning = (r === 'default' ? data.translations.responseDefault : data.translations.responseUnknown);
             var url = '';
             for (var s in common.statusCodes) {
                 if (common.statusCodes[s].code === r) {
-                    entry.meaning = common.statusCodes[s].phrase;
+                    meta.meaning = common.statusCodes[s].phrase;
                     url = common.statusCodes[s].spec_href;
                     break;
                 }
             }
-            if (url) entry.meaning = '[' + entry.meaning + '](' + url + ')';
-            entry.description = (typeof response.description === 'string' ? response.description.trim() : undefined);
-            entry.schema = data.translations.schemaNone;
+            if (url) meta.meaning = '[' + meta.meaning + '](' + url + ')';
+            meta.description = (typeof response.description === 'string' ? response.description.trim() : undefined);
+
             for (let ct in response.content) {
+                let entry = {...meta};
+                entry.status = r;
+                entry.schema = data.translations.schemaNone;
+
                 let contentType = response.content[ct];
-                if (contentType.schema) {
-                    entry.type = contentType.schema.type;
-                    entry.schema = data.translations.schemaInline;
-                }
-                if (contentType.schema && contentType.schema["x-widdershins-oldRef"] && contentType.schema["x-widdershins-oldRef"].startsWith('#/components/')) {
-                    let schemaName = contentType.schema["x-widdershins-oldRef"].replace('#/components/schemas/', '');
-                    entry.schema = '[' + schemaName + '](#schema' + schemaName.toLowerCase() + ')';
-                    entry.$ref = true;
-                }
-                else {
-                    if (contentType.schema && contentType.schema.type && (contentType.schema.type !== 'object') && (contentType.schema.type !== 'array')) {
-                        entry.schema = contentType.schema.type;
+                entry.contentType = ct;
+
+                let ctSchema = contentType.schema;
+                if (ctSchema) {
+                    entry.type = ctSchema.type;
```

Comment:
Superfluous.

I guess is nice having `format` as well but seeing we're using `ctSchema.type` below this point and `entry.originalType` is being set, there is no need to assign `entry.type`.